### ISSUE TITLE
What companies are using this service?

### ISSUE THREAD

**Issue by betweenbrain** [andrieslouw/imagesweserv#174]
This service is perfect for our use, but my employer is wondering what other companies are using and relying on it. Do you happen to have a list? If not, would anyone be willing to share the name(s) of their companies and/or clients using it?

Thanks, and thank you for this great service and project!

**Comment by andrieslouw** [comment]
Due to privacy concerns I'm unable to share names, even if we were able to do so technically.

As stated in [our privacy policy](https://github.com/weserv/images/blob/4.x/Privacy-Policy.md) we only have information from the last 7 days. From this information I see the service is mainly used by ISP's, dealerships, real estate agencies, webshops, social networks and mobile app developers. Our reach is global, and we fetch images from 300.000 different unique domains.

I invite people to share their experiences if they like to.

Be reminded that the service is free, we haven't accepted any (financial) compensation since we started, and we don't affiliate. We encourage people to employ the source-code on their own servers if they want to.

About our providers:
- There is a non-adult content policy, to keep the service from being blocked. Filtering is provided by OpenDNS. We only share origin-domains with them, no specific URL's.
- Cloudflare provides the CDN, all traffic will pass through their network.
- Servers are stationed in France and provided by Scaleway and OVH. In 2017 we've had additional servers in Germany, provided by Hetzner. Before 2014 we had all our servers based in the Netherlands, provided by Netrouting and Datafiber. As for now, we've exclusively used servers within the European Union, and will continu to do so.
